UPGRADE-1.0.md

```diff
@@ -622,6 +622,7 @@
 ### Locale / LocaleBundle
 
 * `Locale` model's `$enabled` field has been removed along with all logic depending on it.
+* The `Locale` documentation now includes avaiable `LocaleContext` methods.
```

Comment:
I am not sure if it is needed. Upgrade file is for describing changes that people need to do in their apps after upgrading from previous versions. Adding docs is cool, although it does not require any changes in the applications.